src/aead/less_safe_key.rs

```diff
@@ -44,6 +44,26 @@ impl LessSafeKey {
         })
     }
 
+    /// Like [open_in_place](Self::open_in_place), except it accepts an arbitrary authentication tag.
+    #[inline]
+    pub fn open_in_place_seperate_tag<'in_out, A>(
+        &self,
+        nonce: Nonce,
+        aad: Aad<A>,
+        tag: Tag,
+        in_out: &'in_out mut [u8],
+    ) -> Result<&'in_out mut [u8], error::Unspecified>
+    where
+        A: AsRef<[u8]>,
+    {
+        let received_tag = tag;
```

Comment:
I think you don't need this line...we can just pass `tag` directly below.

Reply:
That works for me, so I've made the change. This was mostly a stylistic choice.